jerry-core/lit/lit-magic-strings.inc.h

```diff
@@ -17,242 +17,239 @@
 /*
  * List of ECMA magic strings
  *
- * These strings must be ascii strings. If non-ascii strings
- * will be ever needed, a divider will be added to separate
- * ascii and non-ascii groups.
+ * These strings must be ascii strings and needs to be defined in
+ * lexicographical order. If non-ascii strings will be ever needed,
+ * a divider will be added to separate ascii and non-ascii groups.
```

Comment:
But I would also add that NULL character cannot be part of magic strings, because it must be the terminator character of all magic strings.